src/sentry/conf/server.py

```diff
@@ -941,6 +941,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     Queue("nudge.invite_missing_org_members", routing_key="invite_missing_org_members"),
     Queue("auto_resolve_issues", routing_key="auto_resolve_issues"),
     Queue("on_demand_metrics", routing_key="on_demand_metrics"),
+    Queue("integrations_slack_activity_notify", routing_key="integrations_slack_activity_notify"),
```

Comment:
do we need to have ops provision a queue for this?

Reply:
That's a good question, I'm not sure! Because this is part of Sentry and part of self-hosted, would that still be needed, or is it auto created?

Reply:
would ask in the sre channel

Reply:
Looks like it's auto created! 
![Screenshot 2024-04-08 at 3 11 01 PM](https://github.com/getsentry/sentry/assets/5581484/f5a4bc9a-dbbd-423f-bac7-245ce2309c07)